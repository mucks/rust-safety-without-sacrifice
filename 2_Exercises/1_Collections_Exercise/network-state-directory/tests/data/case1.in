# Simple register and count
MEMBER ForestCity 1 Alice Citizen
MEMBER Zuitzerland 2 Bob Citizen
COUNT ForestCity
COUNT Zuitzerland
STATS
