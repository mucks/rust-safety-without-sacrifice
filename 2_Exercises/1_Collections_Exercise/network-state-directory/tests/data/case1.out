COUNT ForestCity: 1
COUNT Zuitzerland: 1
STATS citizens=2 validators=0 founders=0
